sdk/storage/src/blob/blob/block_list.rs

```diff
@@ -73,22 +63,27 @@ where
 #[cfg(test)]
 mod test {
     use super::*;
+    use bytes::Bytes;
 
     #[test]
     fn to_xml() {
         let mut blocks = BlockList { blocks: Vec::new() };
         blocks
             .blocks
-            .push(BlobBlockType::Committed(Vec::from(b"numero1" as &[u8])));
+            .push(BlobBlockType::Committed(Bytes::from_static(b"numero1")));
```

Comment:
Why not use the `BlobBlockType::new_commited` function for this and the other functions for the uses below?